validator.go

```diff
@@ -82,29 +82,28 @@ func (t *TableValidator) validateRow(row []string) error {
 // a set of validators for each field.
 func (t *TableValidator) Init() error {
 	var (
-		err  error
-		head []string
+		err         error
+		head        []string
+		length_err  = false
+		match_err   = false
```

Comment:
Short variable names are idiomatic. If one word variable names are not reasonable, use camelcase.